public/sass/pages/_explore.scss

```diff
@@ -6,6 +6,14 @@
   }
 }
 
+.icon-brand-gradient {
```

Comment:
As icon-brand-gradient is used at the multiple places across Explore, I have created class for it.